pkg/operator/ceph/object/objectstore.go

```diff
@@ -289,7 +290,11 @@ func createSimilarPools(context *Context, pools []string, poolSpec model.Pool) e
 			if err != nil {
 				return errors.Errorf("failed to create pool %s for object store %s", name, context.Name)
 			}
+		} else {
+			ceph.SetPoolProperty(context.Context, context.ClusterName, name, "size", strconv.Itoa(int(poolDetails.Size)))
 		}
+		//update the pools if it already exist
```

Comment:
shouldn't this comment be inside the `else` statement?

Reply:
yep it should be :) , I’ll make it at the start of else block

Reply:
Done!, Closing this conversation.